src/components/renderer/file-upload.vue

```diff
@@ -95,7 +97,7 @@ export default {
 
     this.removeDefaultClasses();
     
-    this.checkIfInRecordList();
+    // this.checkIfInRecordList();
```

Comment:
@eiresendez Can you remove this and the code associated with this method since it is no longer being used?

Reply:
sure, done! @sanjacornelius